spec/requests/graphql/query/digital_object_import_spec.rb

```diff
@@ -20,12 +20,12 @@
           {
             "batchImport": {
               "digitalObjectImport": {
-                "createdAt": "#{digital_object_import.created_at.strftime('%FT%TZ')}",
+                "createdAt": "#{digital_object_import.created_at.to_datetime.strftime('%FT%T%Z')}",
```

Comment:
```suggestion
               # use DateTime's strftime implementation over ActiveSupport::TimeWithZone's
                "createdAt": "#{digital_object_import.created_at.to_datetime.strftime('%FT%T%Z')}",
```

Reply:
```suggestion
                # converting ActiveSupport::TimeWithZone to DateTime so that %Z outputs hour offset instead of time zone name
                "createdAt": "#{digital_object_import.created_at.to_datetime.strftime('%FT%T%Z')}",
```